cloudformation/netagent2/banyan-access-tier-v2-amazon-linux.json

```diff
@@ -0,0 +1,439 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "Banyan Elastic Access Tier v2",
+  "Metadata": {
+    "AWS::CloudFormation::Interface": {
+      "ParameterGroups" : [
+        {
+          "Label": { "default": "Banyan Configuration" },
+          "Parameters": ["ApiKeySecret", "CommandCenterURL", "BanyanPackage"]
+        },
+        {
+          "Label": { "default" : "Network Configuration" },
+          "Parameters": ["VPCID", "PublicSubnets", "PrivateSubnets", "HealthCheckCidr", "CrossZoneEnabled"]
+        },
+        {
+          "Label": {"default": "Instance Configuration"},
+          "Parameters": ["KeyName", "DatadogApiKey", "MinInstances", "DefaultAmiId", "AmiId", "ManagementCidr"]
+        }
+      ],
+      "ParameterLabels": {
+        "VPCID": { "default": "Which VPC should this Access Tier be deployed to?" }
+      }
+    }
+  },
+  "Parameters": {
+    "VPCID": {
+      "Description": "VPC must have internet access through an Internet Gateway",
+      "Type": "AWS::EC2::VPC::Id"
+    },
+    "PublicSubnets": {
+      "Description": "Public subnets for the load balancer.",
+      "Type": "List<AWS::EC2::Subnet::Id>"
+    },
+    "PrivateSubnets": {
+      "Description": "Private subnets for the instances.",
+      "Type": "List<AWS::EC2::Subnet::Id>"
+    },
+    "MinInstances": {
+      "Default": "2",
+      "Description": "Minimum number of instances. 1 instance per az (subnet) recommended",
+      "Type": "String"
+    },
+    "CrossZoneEnabled": {
+      "Default": "false",
+      "Description": "Enable to distribute traffic across all available instances",
+      "Type": "String",
+      "AllowedValues" : ["true", "false"]
+    },
+    "ApiKeySecret": {
+      "Description": "Create an API Key in the Admin Settings section of the Banyan console. Alternatively, create a Refresh Token in the 'My Profile' section of the Banyan console",
```

Comment:
Indicate that minimum privilege should be `access_tier` scope?

Also, Tarun had indicated that we should deprecate the refresh token for access tiers. Maybe that's another conversation, though.

Reply:
done